services/movie.py

```diff
@@ -25,16 +30,17 @@ def get_movie_by_id(movie_id: int) -> Movie:
 def create_movie(
     movie_title: str,
     movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None,
```

Comment:
Using mutable default arguments like `list` can lead to unexpected behavior. Consider using `None` as the default and initializing the list inside the function.